resources/services/ec2/images.go

```diff
@@ -7,18 +7,20 @@ import (
 	"github.com/aws/aws-sdk-go-v2/service/ec2"
 	"github.com/aws/aws-sdk-go-v2/service/ec2/types"
 	"github.com/cloudquery/cq-provider-aws/client"
+
 	"github.com/cloudquery/cq-provider-sdk/provider/schema"
 )
 
 func Ec2Images() *schema.Table {
 	return &schema.Table{
-		Name:         "aws_ec2_images",
-		Description:  "Describes an image.",
-		Resolver:     fetchEc2Images,
-		Multiplex:    client.ServiceAccountRegionMultiplexer("ec2"),
-		IgnoreError:  client.IgnoreAccessDeniedServiceDisabled,
-		DeleteFilter: client.DeleteAccountRegionFilter,
-		Options:      schema.TableCreationOptions{PrimaryKeys: []string{"account_id", "id"}},
+		Name:          "aws_ec2_images",
+		Description:   "Describes an image.",
+		Resolver:      fetchEc2Images,
+		Multiplex:     client.ServiceAccountRegionMultiplexer("ec2"),
+		IgnoreError:   client.IgnoreAccessDeniedServiceDisabled,
+		DeleteFilter:  client.DeleteAccountRegionFilter,
+		Options:       schema.TableCreationOptions{PrimaryKeys: []string{"account_id", "id"}},
+		IgnoreInTests: true,
```

Comment:
just double verifying if we don't have an image in our test infrastructure

Reply:
`    images_test.go:12: expected to have at least 1 entry at table aws_ec2_images got zero` if I remove it.